eng/Version.Details.xml

```diff
@@ -63,5 +63,9 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>044fa99d22e576fcfaf63f09ee9bba821855dddb</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.NETCore.Platforms" Version="3.0.0-preview.19073.11">
```

Comment:
Will we just have to remember to update this and https://github.com/aspnet/Extensions/pull/1021/files#diff-8b8f08ffbf7b863fb3700c1718eeb4cbR19 simultaneously? Seems perilous&hellip;

Reply:
The bot should take care of updating both files.